CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- `selectedSla` being reseted to a default value.
```

Comment:
nit: The past participle of _to reset_ is _reset_.
Also, could you be more specific about the function and conditions in which the fix was applied? A person reading this changelog would little clue of what this fix is about.

Reply:
Sure!